osu.Game/Online/Chat/ChannelManager.cs

```diff
@@ -256,8 +256,38 @@ public void PostCommand(string text, Channel target = null)
                     JoinChannel(channel);
                     break;
 
+                case "msg":
+                    if (string.IsNullOrWhiteSpace(content))
+                    {
+                        target.AddNewMessages(new ErrorMessage("Usage: /msg [user] [msg]"));
+                        break;
+                    }
+
+                    var args = content.Split(" ");
+
+                    var request = new GetUserRequest(args[0]);
+                    request.Success += user =>
+                    {
+                        OpenPrivateChannel(user);
+                        var message = new Message
+                        {
+                            Content = string.Join(" ", args[1..]),
+                            Sender = user,
+                            IsAction = false,
+                            ChannelId = CurrentChannel.Value.Id,
+                            Timestamp = DateTimeOffset.Now,
+                            DisplayContent = string.Join(" ", args[1..])
+                        };
```

Comment:
```suggestion
                    var args = content.Split(' ', 2);

                    var request = new GetUserRequest(args[0]);
                    request.Success += user =>
                    {
                        OpenPrivateChannel(user);
                        var message = new Message
                        {
                            Content = args[1],
                            Sender = user,
                            IsAction = false,
                            ChannelId = CurrentChannel.Value.Id,
                            Timestamp = DateTimeOffset.Now,
                            DisplayContent = args[1],
                        };
```
It's probably better not to oversplit and then rejoin.

Reply:
Makes sense, I've updated the PR with the changes.